### ISSUE TITLE
Create gifs with folder structure after AutoML training

### ISSUE THREAD

**Issue by pplonski** [mljar/mljar-supervised#221]
Create gifs with folder structure after AutoML training and add them to the Readme

**Comment by aakarsh1011** [comment]
Hi, I want to work on this feature, and I want to get more info on it. By the way, I'm new to the open-source world, so please bear with me.

**Comment by pplonski** [comment]
Hi @aakarsh1011,

Great, happy to see you here. This issue is to improve docs/readme.

After running MLJAR AutoML there is produced a results directory with many files. I was thinking about making a GIF that will show simple training (for example on Titanic data which is well-known) and then show the output files and output README.md reports. The plan for such GIF can be:
1. Show code for training on Titanic data.
2. Show output from training.
3. Show results directory with files.
4. Present README.md files (display reports).

The final GIF will be some kind of video/presentation that will showcase the features of MLJAR AutoML. The final GIF will be added to the package README.

@aakarsh1011 what do you think?

**Comment by aakarsh1011** [comment]
Hey @pplonski, so I have the results folder and I wanted to all the requirements mentioned above should be in 1 single gif ?

**Comment by pplonski** [comment]
@aakarsh1011 yes, the gif should show all good things that are created with the package.

**Comment by aakarsh1011** [comment]
I have uploaded the gif on mljar/mljar-examples and created a pull request their take a look and tell me if it required changes and I will do the needful.

**Comment by pplonski** [comment]
@aakarsh1011 I like GIF very much. Great job. I've added it to the package readme. You can check it here https://github.com/mljar/mljar-supervised#demo - feel free to add more description to the readme.

What software have you used for GIF creation?

Is this your Twitter account https://twitter.com/yadavaakarsh? I will ask @aplonska to tweet about your great demo.

Thank you!

**Comment by aakarsh1011** [comment]
@pplonski Thank you so much and yes that is my Twitter handle.
I am using a software called "LICEcap"
